### PR TITLE
fix(android): 279 - Fix error cannot find symbol

### LINKED ISSUE #279: Build failed. Error cannot find symbol builder.setNavigationBarColor
```
> Task :react-native-inappbrowser-reborn:compileReleaseJavaWithJavac FAILED
/Users/alexandrsmirnov/Work/E-Orda/Colibri-mobile/node_modules/react-native-inappbrowser-reborn/android/src/main/java/com/proyecto26/inappbrowser/RNInAppBrowser.java:107: error: cannot find symbol
        builder.setNavigationBarColor(Color.parseColor(colorString));
               ^
  symbol:   method setNavigationBarColor(int)
  location: variable builder of type Builder
/Users/alexandrsmirnov/Work/E-Orda/Colibri-mobile/node_modules/react-native-inappbrowser-reborn/android/src/main/java/com/proyecto26/inappbrowser/RNInAppBrowser.java:116: error: cannot find symbol
        builder.setNavigationBarDividerColor(Color.parseColor(colorString));
               ^
  symbol:   method setNavigationBarDividerColor(int)
  location: variable builder of type Builder
2 errors
```
### Which platform(s) does your issue occur on?
- Android 

**Package.json**
```
"react-native": "^0.64.2"  
"react-native-inappbrowser-reborn": "^3.6.1" 
```

### PR DESCRIPTION
**Pull request by jdnichollsc**
Fix `Build failed. Error cannot find symbol builder.setNavigationBarColor` error for Android Support

## PR Checklist

- [x] The PR title follows our guidelines: https://github.com/proyecto26/react-native-inappbrowser/blob/main/CONTRIBUTING.md#pull-request-process.
- [x] There is an issue for the bug/feature this PR is for. To avoid wasting your time, it's best to open a suggestion issue first and wait for approval before working on it.
- [x] All existing tests are passing
- [ ] Tests for the changes are included

## What is the current behavior?

Android Support is not working with the new methods of `CustomTabsIntent.Builder` for `AndroidX`.

## What is the new behavior?

Detect if the method exists before to invoke the function to avoid build issues by using Reflection.

Fixes/Implements/Closes #279.

### CODE REVIEW

**Comment by reberthkss** [comment]
@jdnichollsc i think it should work, let me try it

**Comment by jdnichollsc** [comment]
@reberthkss awesome, thanks mate! <3

**Comment by jdnichollsc** [comment]
Personally I don't like Reflection, but I can't see any other option 🤔

**Comment by reberthkss** [comment]
I've tested on my project and the error was gone, but I faced the same problem with `create_react_context` described in https://github.com/proyecto26/react-native-inappbrowser/issues/279#issuecomment-873325797

I'm trying to test in a bare react-native typescript template project

**Comment by jdnichollsc** [comment]
Ohh very odd, please let me know! 🙏

**Comment by reberthkss** [comment]
Worked like a charm :3

https://user-images.githubusercontent.com/54030471/124369100-96bd3f00-dc3e-11eb-98a2-9f7917801af5.mp4

File: android/src/main/java/com/proyecto26/inappbrowser/RNInAppBrowser.java [hunk 1]
```diff
@@ -62,6 +63,26 @@
   private Activity currentActivity;
   private static final Pattern animationIdentifierPattern = Pattern.compile("^.+:.+/");
 
+  public Integer setColor(CustomTabsIntent.Builder builder, final ReadableMap options, String key, String method, String colorName) {
+    String colorString = "";
```
Comment (reberthkss) [inline android/src/main/java/com/proyecto26/inappbrowser/RNInAppBrowser.java:67]: can be `String colorString = options.getString(key);`?

and further on try/catch block : 

```
String colorString = options.getString(key);

...

try {
if (colorString != null) {
...
}

...
```

**Comment by jdnichollsc** [comment]
@reberthkss nice! thanks for your help! 🤩

File: android/src/main/java/com/proyecto26/inappbrowser/RNInAppBrowser.java (hunk 1 above)
Comment (jdnichollsc) [reply android/src/main/java/com/proyecto26/inappbrowser/RNInAppBrowser.java:67]: We need to validate if the key exist first, using `options.hasKey(key)`

Comment (jdnichollsc) [reply android/src/main/java/com/proyecto26/inappbrowser/RNInAppBrowser.java:67]: @reberthkss did you find another bug?

Comment (reberthkss) [reply android/src/main/java/com/proyecto26/inappbrowser/RNInAppBrowser.java:67]: just for known more, the response from `getString` function is Nullable, so if the key doesn't exist the `colorString` variable was null, alright? Why don't follow this approach? Is it a bad practice/insecure?

Comment (jdnichollsc) [reply android/src/main/java/com/proyecto26/inappbrowser/RNInAppBrowser.java:67]: Oh no, this validation is old, if you want I can change it, no problem, thanks for noticing!

Comment (jdnichollsc) [reply android/src/main/java/com/proyecto26/inappbrowser/RNInAppBrowser.java]: Done!

**Review by reberthkss — APPROVED** [review]

**Comment by pramahaditamaputra-ralali** [comment]
The new release version 3.6.2 its works as well right Now in my app built with React Native version 0.61.5 , thanks for the fixing so i can use the lastest version of this library @jdnichollsc @reberthkss  <3

**Comment by thuongtv-vn** [comment]
Hi All,

From my point of view, The bars' color should be optional. What do you think?

**Comment by thuongtv-vn** [comment]
Hi

I founded an issue on the latest version 3.6.2 with:

**NoSuchKeyException**

react-native: 0.60.5

The issue is not happening for example which is using react-native 0.64.2. 

I have created a pull request:

https://github.com/proyecto26/react-native-inappbrowser/pull/283

**Pull request merged**